test/unit/test_boundary_conditions.py

```diff
@@ -543,15 +543,23 @@ def test_string_for_field_in_dirichletbc():
     bc.create_dirichletbc(V, fenics.Constant(1), surface_marker)
 
 
-def test_dissociation_flux_solve():
-    """Test to catch bug DissociationFlux see #581"""
+@pytest.mark.parametrize(
+    "boundary",
+    [
+        (festim.DissociationFlux(surfaces=[1], Kd_0=1, E_Kd=0, P=1e4)),
+        (festim.ConvectiveFlux(h_coeff=1, T_ext=1, surfaces=1)),
+        (festim.FluxBC(surfaces=1, value=1, field=0)),
+        (festim.MassFlux(h_coeff=1, c_ext=1, surfaces=1)),
+        (festim.RecombinationFlux(Kr_0=1e-20, E_Kr=0, order=2, surfaces=1)),
```

Comment:
We could be even more thorough here and add some Dirichlet boundary conditions to this list!